resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -430,6 +433,7 @@ private[spark] class KubernetesClusterSchedulerBackend(
       .endMetadata()
       .withNewSpec()
         .withHostname(hostname)
+        .withNodeSelector(nodeSelector.asJava)
```

Comment:
@kimoonkim was this the k8s 1.6+ feature you were referencing in https://github.com/apache-spark-on-k8s/spark/pull/316/files#diff-206ce9343d722622e995071cbb69c330R338 ?